### PR TITLE
Added possibility to change default directory (Fixes #134)

### LINKED ISSUE #134: Add possibility to change default directory
Right now we always use `/opt/snare/` for storing all the snare files like config and uuid. It might be good to add the possibility to change this folder.

### PR DESCRIPTION
**Pull request by CyberDrudge**
Fixes #134 
This adds the possibility to change the default directory which is currently set to `/opt/snare`. Now, users can change their default path by using a `--path <directory_path>` argument.

### CODE REVIEW

**Comment by afeena** [comment]
@CyberDrudge Hi, what is the difference between this pr and #162 ? :)

File: bin/clone [hunk 1]
```diff
@@ -38,15 +38,25 @@ def main():
     parser.add_argument(
         "--css-validate", help="set whether css validation is required", type=str_to_bool, default=None
     )
+    parser.add_argument("--path", help="path to save the page to be cloned", required=False,default=None)
     args = parser.parse_args()
+    if args.path:
+        default_path = os.path.join(args.path, 'snare')
+    else:
+        default_path = '/opt/snare'
```
Comment (rnehra01) [inline bin/clone]: Set it in the argument itself.

**Comment by rnehra01** [comment]
@CyberDrudge Please make the changes.

**Comment by rnehra01** [comment]
@CyberDrudge ??

**Comment by Aniket21mathur** [comment]
@rnehra01 I have made the required changes in #226. Please review.

**Comment by afeena** [comment]
@Aniket21mathur please resolve conflicts

**Pull request closed without merging**